bindings/profilers/wall.cc

```diff
@@ -291,6 +291,9 @@ void SignalHandler::HandleProfilerSignal(int sig,
     return;
   }
   auto isolate = Isolate::GetCurrent();
+  if (!isolate || isolate->IsDead()) {
```

Comment:
In which cases does `GetCurrent` returns null ?

Reply:
I honestly don't know – I'm just trying to be pedantically defensive and not unconditionally dereference a pointer. I can envision a race condition between signal delivery and an isolate going away? I know the logic is different, but even the V8 sampler manager [checks](https://github.com/v8/v8/blob/0f07e2d2a594b77b98b6596d4597f25de1e6c53c/src/libsampler/sampler.cc#L257) for isolate on a sampler not being null.

Reply:
I think it's ok as this.